src/main/kotlin/jambda/camp/clean/domain/Product.kt

```diff
@@ -0,0 +1,14 @@
+package jambda.camp.clean.domain
+
+data class Product(
+    val name: ProductName,
+    val price: ProductPrice
+)
+
+data class ProductName(
+    val value: String
```

Comment:
Minor: Namensgebung -> Ich finde entweder sollten bei Domain Primitives die Werte darin alle ```value``` heissen wie hier oder einen sprechenden Namen haben wie ```name``` in diesem Fall. Was denkt ihr?